Jumpscale/clients/blockchain/tfchain/TFChainCapacity.py

```diff
@@ -325,7 +328,7 @@ def _process_reservation(self, reservation, threebot_id, source=None, refund=Non
 
         # validate bot id exists
         bot = self._wallet.client.threebot.record_get(threebot_id)
-        reservation_expiry = j.tools.time.extend(time.time(), reservation.duration)
+        reservation_expiry = j.clients.tfchain.time.extend(time.time(), reservation.duration)
```

Comment:
Did you already removed the `j.tools.time` ?

Reply:
I didn't since it has different logic and has tests that cover it, might come in handy to someone else, no?

Reply:
ok